### PR TITLE
Use aside-layout

### PR DESCRIPTION
**Pull request by restlessbit**
Updates the server template to use a sidebar layout instead of throwing all sections onto a single page. This makes the styleguide a little bit easier to consume and navigate around.

screen shot 2016-06-08 at 7 40 15 pm

/cc @underdogio/engineering

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -38,7 +38,31 @@ module.exports = function(cb) {
...
+      var currentSection = sections.filter(function findSection (section) {
+        // Assign the isActive property for each section to change link styling
+        // for active vs non-active sections in the sidebar
+        section.isActive = (section.slug === sectionSlug);
+        return section.slug === sectionSlug;
+      })[0];
+
+      if (!currentSection) {
+        // Return a 404 if no section with a matching slug was found
+        return res.sendStatus(404);
```
Comment (brettlangdon) [inline server/index.js]: Should probably add a ticket to add a real 404 page template in here.

File: server/views/styleguide.hbs [hunk 2]
```diff
@@ -0,0 +1,38 @@
+<div class="app-container">
+  <div class="app-container__header header border--bottom">
+    <a class="push25--left" href="/">Underdog.io</a>
```
Comment (brettlangdon) [inline server/views/styleguide.hbs]: No fancy logo? :)

**Comment by brettlangdon** [comment]
lgtm, I like the direction.

File: server/index.js (hunk 1 above)
Comment (restlessbit) [inline server/index.js]: Yeah

File: server/views/styleguide.hbs (hunk 2 above)
Comment (restlessbit) [inline server/views/styleguide.hbs]: Not yet :)

**Pull request merged**